src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -131,7 +131,7 @@
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         c.writeLog(ws, baos);
         assertTrue(c.exitStatus(ws, launcher).intValue() != 0);
-        assertThat(baos.toString(), containsString("Bogus error"));
+        assertTrue(baos.toString().contains("Bogus error"));
```

Comment:
Any reason for this change? The previous assertion with `assertThat` has the benefit of showing the expected and actual strings if the assertion failed, whereas the new assertion will just say something like "expected true, got false", which is a lot less useful.

Reply:
No, I was changing things up because that assertion was failing on ci.jenkins.io, but in the end it turned out to be a timeout issue with our build servers.  I'll go ahead and change it back.